e2e/dractions/disapp.go

```diff
@@ -151,3 +159,18 @@ func failoverRelocateDiscoveredApps(
 
 	return deployers.WaitWorkloadHealth(ctx, drCluster, appNamespace)
 }
+
+// createNamespaces sets up namespaces for discovered app protection based on the Kubernetes distribution.
+// For Kubernetes, creates namespaces and adds annotation on both DR clusters for volsync based replication.
+// For OpenShift, creates namespace only on the target DR cluster (c2) before failover.
+// Returns an error if the distribution is unknown, and if namespace creation or annotation fails.
+func createNamespaces(ctx types.Context, namespace string, log *zap.SugaredLogger) error {
```

Comment:
Should be createNamespacesDiscoveredApps so next developer will not try to use it from managed apps actions. The name is ugly but it matches other functions in this file.